src/less/flag/flag.less

```diff
@@ -1,8 +1,13 @@
-svg.flag {
-    display: inline-block;
-    max-height: 48px;
-    max-width: 64px;
-    pointer-events: none;
-    stroke-width: 0;
-    vertical-align: middle;
+svg {
```

Comment:
This seems a bit out of scope for this issue. Changes in `.less` files of other components is usually code smell for scope issues. 

I can see the value of doing this in `flag`, but there are other options too, such as cascading from disabled elements, such as `<input disabled>` or `<button disabled>`. That should probably be in its own issue with docs and stories to cover the disabled variations either for flag alone or cascading from `[disabled]`. 

For now, let's just cascade down inside this component to stay in scope for the work. This approach will also require less internal markup (no disabled flag class on the svg itself):

```css
.phone-input--disabled svg { ... }
```